packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -279,7 +285,28 @@ <h2>Meetings Devices</h2>
       <form id="current-device">
         <fieldset>
           <legend>Current Device</legend>
-          <label>[[CURRENT DEVICE IDENTIFIER && DETAILS]]</label>
+          <label>Device Details</label>
+          <pre id="current-device-details">No Device Selected</pre>
+          <label>Device Pairing</label>
+          <br />
+          <button id="device-request-pin" type="button" onclick="webex.devicemanager.requestPin(currentDevice)">webex.devicemanager.requestPin()</button>
+          <input id="device-pin-query" name="devicePinQuery" placeholder="PIN" value="" type="number" />
+          <button id="device-request-pair" type="button" onclick="deviceRequestPair()">webex.devicemanager.pair()</button>
+          <button id="device-request-unpair" type="button" onclick="deviceRequestUnpair()">webex.devicemanager.unpair()</button>
+          <br />
+          <label id="device-pair-state">Not paired</label>
+          <br />
+          <br />
+          <label>Device Audio State</label>
```

Comment:
We could mention here that the state isn't 'live' and will need to be updated by hitting the 'deviceGetAudioState' button

Reply:
Good idea, I'll proceed with making this change now.